File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L2/01/L2.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter the radius and length of a cylinder: ', (input) => {
  const [jari2, panjang] = input.split(' ').map(Number);
  const luas = jari2 * jari2 * 3.14159;
  const total = luas * panjang;

  console.log(`The area is ${luas}`);
  console.log(`The volume of the cylinder is ${total}`);

  rl.close();
});

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-02/plagiarized/L2/01/L2.kt
package IR

import java.util.*

object L2 {
    @JvmStatic
    fun main(args: Array<String>) {
        val input = Scanner(System.`in`)

        print("Enter the radius and length of a cylinder: ")
        val jari2 = input.nextDouble()
        val panjang = input.nextDouble()
        val luas = jari2 * jari2 * 3.14159
        val total = luas * panjang
        println("The area is $luas")
        println("The volume of the cylinder is $total")
    }
}
